setup.py

```diff
@@ -2,11 +2,19 @@
 # -*- coding: utf-8 -*-
 
 from setuptools import setup
+from os import path
 import re
 import os
 import sys
 
 
+with open(
+        f"{path.abspath(path.dirname('__file__'))}/README.md",
+        encoding='utf-8',
+        ) as f:
+    long_description = f.read()
```

Comment:
```suggestion
        ) as file:
    long_description = file.read()
```

I know it may seem silly but maybe it is more readable "file" than "f"